x-pack/plugins/ml/server/types.ts

```diff
@@ -12,12 +12,17 @@ import type { SecurityPluginSetup } from '../../security/server';
 import type { PluginSetupContract as FeaturesPluginSetup } from '../../features/server';
 import type { LicensingPluginSetup } from '../../licensing/server';
 import type { SpacesPluginSetup, SpacesPluginStart } from '../../spaces/server';
+
 import type { MlLicense } from '../common/license';
 import type { ResolveMlCapabilities } from '../common/types/capabilities';
 import type { RouteGuard } from './lib/route_guard';
 import type { AlertingPlugin } from '../../alerting/server';
 import type { ActionsPlugin } from '../../actions/server';
 import type { UsageCollectionSetup } from '../../../../src/plugins/usage_collection/server';
+import type {
+  PluginSetup as PluginSetupData,
```

Comment:
Seems like `DataPluginSetup` and `DatePluginStart` would align better with the other names and make more sense.

Reply:
Renamed here https://github.com/elastic/kibana/pull/99905/commits/034c680090836471795efc4bf8e30f0db5806939